File: Java/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        int n;
        System.out.println("Enter the number: ");
        n = input.nextInt();
        System.out.println(Fact(n));
    }

    public static int Fact(int n){
        if(n <= 1)
            return 1;
        else{
            return n * Fact(n - 1);
        }
    }
}

File: R/Factorial.r
print("Enter the number: ")
num <- readline()
num <- as.numeric(num)
factorial <- 1

fact <- function(num){
  if(num < 0){
    print("Factorial of a negative number doesn't exist")
  }else{
    for(i in 1:num){
      factorial <- factorial * i
    }
  }
  print(factorial)
}

fact(num)